src/core/packages/chrome/layout/core-chrome-layout-components/layout_global_css.tsx

```diff
@@ -80,10 +82,17 @@ export const LayoutGlobalCSS = () => {
   `;
 
   const application = css`
+    ${layoutVarName('application.marginBottom')}: ${applicationMarginBottom}px;
```

Comment:
nit: should we go ahead and add margin top and left as 0, for completeness and flexibility?